freqtrade/freqai/data_drawer.py

```diff
@@ -568,12 +567,16 @@ def load_data(self, coin: str, dk: FreqaiDataKitchen) -> Any:
         # try to access model in memory instead of loading object from disk to save time
         if dk.live and coin in self.model_dictionary:
             model = self.model_dictionary[coin]
-        elif not dk.keras:
+        elif self.model_type == 'joblib':
             model = load(dk.data_path / f"{dk.model_filename}_model.joblib")
-        else:
+        elif self.model_type == 'keras':
             from tensorflow import keras
-
             model = keras.models.load_model(dk.data_path / f"{dk.model_filename}_model.h5")
+        elif self.model_type == 'stable_baselines':
+            mod = __import__('stable_baselines3', fromlist=[
```

Comment:
I think we should stick to one way to import a module.

you use `__import__()` here - but `importlib.import_module` in the BaseReinforcementModel - which i think is the "cleaner" way to do dynamic imports (it's not using an internal method directly).

Reply:
Ah, missed that one. Fixing it.